frontend/src/App.vue

```diff
@@ -149,6 +149,13 @@ import { RouterLink, RouterView } from "vue-router";
           </ul>
         </aside>
       </div>
+      <div class="mt-auto mr-auto ml-auto">
```

Comment:
div is located in the bottom (I like that). If I make it smaller (height), the about license is too close. Could you increase margin / padding?
![image](https://github.com/eclipse-tractusx/puris/assets/116013795/4cfc314b-ae19-4606-8dc6-d9066432db7e)